frontend/src/App.js

```diff
@@ -197,6 +278,12 @@ class App extends React.Component {
           top={city.top}
           left={city.left}
           addRemoveSelectedCity={this.addRemoveSelectedCity}
+          disabled={disabled}
```

Comment:
nit: recommend a name change for this property. Maybe something like `disabledClass`